src/visual.ts

```diff
@@ -84,16 +82,19 @@ import {
     SubSelectableDisplayNameAttribute,
     SubSelectableObjectNameAttribute
 } from "powerbi-visuals-utils-onobjectutils"
+
+// d3
+import { select, Selection as d3Selection } from "d3-selection";
```

Comment:
please mention d3 in import naming. `import { select as d3Select, ...}`

Reply:
Done.